strings.txt

```diff
@@ -16015,6 +16015,36 @@
     ja = ナビゲーション中は更新できません。
     vi = Không cho phép cập nhật trong khi điều hướng.
 
+  [migration_no_space_message]
```

Comment:
Там другой текст, Игорь сказал что нужна именно эта строка.
> On Apr 29, 2016, at 2:05 PM, Alexander Marchuk <notifications@github.com> wrote:
> 
> In strings.txt <https://github.com/mapsme/omim/pull/3075#discussion_r61561877>:
> 
> > @@ -16015,6 +16015,36 @@
> >      ja = ナビゲーション中は更新できません。
> >      vi = Không cho phép cập nhật trong khi điều hướng.
> >  
> > +  [migration_no_space_message]
> Есть строка downloader_no_space_message. Видимо, надо использовать этот ключ.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub <https://github.com/mapsme/omim/pull/3075/files/1c95636bb154b14b480bf83f416967a6b8ef1a43#r61561877>